### PR TITLE
The client code gets access to the Hammerhead script (close #479)

### LINKED ISSUE #479: The client code gets access to the Hammerhead script
Client code:

```javascript 
div.innerHTML = '<html></html>';
```

After that `div.childNodes` contains hammerhead service script.

See https://isc.devexpress.com/Thread/WorkplaceDetails/T352264

### PR DESCRIPTION
**Pull request by churkin**
\cc @kirovboris @LavrovArtem @VasilyStrelyaev

### CODE REVIEW

**Comment by kirovboris** [comment]
lgtm

**Comment by LavrovArtem** [comment]
lgtm

File: src/client/utils/html.js [hunk 1]
```diff
@@ -47,49 +48,59 @@ export function isPageHtml (html) {
     return /^\s*(<\s*(!doctype|html|head|body)[^>]*>)/i.test(html);
 }
 
-function processPageTag (pageTagHtml, process) {
-    pageTagHtml = pageTagHtml.replace(/^(\s*<\s*)(head|body|html)/i, '$1fakeTagName_$2');
+function spoofPageTag (tagHtml) {
```
Comment (VasilyStrelyaev) [inline src/client/utils/html.js]: spoofPageTag -> hidePageTag ??

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by churkin** [comment]
FPR

**Comment by VasilyStrelyaev** [comment]
lgtm

**Pull request merged**